docker-compose/.env

```
@@ -1,5 +1,77 @@
+# ---------------------------------------------------------------------------
 INFLUXDB_TOKEN=apiv3_suivi-bourse-admin-token-for-local-development
+INFLUXDB_DATABASE=suivi_bourse
+
+# ---------------------------------------------------------------------------
+# Deployment
+# ---------------------------------------------------------------------------
+# App image tag. Pin an exact version (e.g. 4.2.1) for reproducible upgrades
+# and rollbacks; a bare major tracks the latest release of that major.
+SB_VERSION=4
+# Config directory mounted into the app. Copied from data.example/.
+SB_CONFIG_DIR=./data
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SB_CONFIG_DIR key should go before the SB_VERSION key

(UnorderedKey)